### DISCUSSION TITLE
Is there an API to query all entities around a position in special radius?

### DISCUSSION THREAD

**Discussion by cmanlh** [bevyengine/bevy#10005] [Q&A]
I'm learning ECS. And need query all other entities around target entity.

Through loop all entities and check their position is in the circle or not, which can work. But if there exist API, I think that will be better.

Thanks.

**Comment by 5tr1k3r** [comment]
Someone correct me if I'm wrong, but I don't think such an API exists. You might be interested in this, though: https://github.com/laundmo/bevy-spatial

**Comment by rlidwka** [comment]
I'm doing that with [rstar](https://github.com/georust/rstar), maintaining list of entities and their positions in a separate rtree.

**Comment by cmanlh** [comment]
Thank you.
